src/components/patterns/elements/select-list.tsx

```diff
@@ -116,7 +117,7 @@ const SelectList = ({options, label, multiple, ariaLabelledby, ...props}: Props)
     >
       <button
         {...getButtonProps()}
-        className="w-full rounded-3xl border border-black-40 p-5 pl-15 text-left"
+        className={"w-full border border-black-40 px-5 py-9 pl-15 text-left " + (optionChosen ? "rounded-3xl" : "rounded-full")}
```

Comment:
@pookmish I missed something. It doesn't work. I think because I left in: `" + (optionChosen ? "rounded-3xl" : "rounded-full")}`. When I remove it, the rounded doesn't appear anyway.

Reply:
Looks to be working for me just fine 
![Uploading image.png…]()

Reply:
Yes, those ones work.  I wanted the homepage hours one to have a rounded-full but it still picks up the round-3xl.  
<img width="358" alt="Screenshot 2024-07-26 at 2 51 08 PM" src="https://github.com/user-attachments/assets/1451063b-b6db-4c28-9846-299340552fa4">

Reply:
There's an option chosen at all times on the home page. So that's how it's going to work. IMO it's correct in this way because when you choose the library with a long name.
![image](https://github.com/user-attachments/assets/a371ea51-56fb-4961-b481-de7f45d113e0)

Reply:
Ok. I think it looks nice too.